src/shared/json.c

```diff
@@ -1066,6 +1066,52 @@ double json_variant_real(JsonVariant *v) {
         return 0.0;
 }
 
+int json_variant_bytes(JsonVariant *v, uint8_t **ret, size_t *ret_size) {
+        _cleanup_free_ uint8_t *buf = NULL;
+        size_t n;
+
+        assert(ret);
+        assert(ret_size);
+
+        if (!v)
+                goto not_found;
+        if (!json_variant_is_array(v))
+                return -EINVAL;
+
+        n = json_variant_elements(v);
+
```

Comment:
Please drop the empty line. That is, handle values soon after the related function call.